### PR TITLE
Fix error message on drop of non-existing MV

### PR DESCRIPTION
**Pull request by raunaqmorarka**

### CODE REVIEW

File: plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java [hunk 1]
```diff
@@ -146,6 +146,15 @@ public void testSessionCatalogSchema()
         assertUpdate(session, "DROP MATERIALIZED VIEW iceberg.tpch.materialized_view_session_test");
     }
 
+    @Test
+    public void testDropIfExists()
```
Comment (findepi) [inline plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java:150]: add a test with Hive too.

utilmately, we shouild have `BaseConnectorTest` coverage for materialized views. cc @sopel39 @losipiuk

Comment (raunaqmorarka) [reply plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java:150]: Hive connector doesn’t support MVs.
I'll look into adding BaseConnectorTest coverage for materialized views in future.

Comment (findepi) [reply plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java:150]: Thanks, let's maybe have an issue for `BaseConnectorTest` coverage.

Comment (raunaqmorarka) [reply plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java:150]: Created https://github.com/trinodb/trino/issues/8054

File: core/trino-main/src/main/java/io/trino/execution/DropMaterializedViewTask.java [hunk 2]
```diff
@@ -57,7 +57,7 @@ public String getName()
         Optional<ConnectorMaterializedViewDefinition> view = metadata.getMaterializedView(session, name);
         if (view.isEmpty()) {
             if (!statement.isExists()) {
-                throw semanticException(TABLE_NOT_FOUND, statement, "View '%s' does not exist", name);
+                throw semanticException(TABLE_NOT_FOUND, statement, "Materialized view '%s' does not exist", name);
```
Comment (sopel39) [inline core/trino-main/src/main/java/io/trino/execution/DropMaterializedViewTask.java:60]: nit: we could also have a engine only test (like we do for selects, see: `TestAnalyzer`)

**Review by sopel39 — APPROVED** [review]

**Comment by sopel39** [comment]
@findepi do you have other comments?

**Comment by findepi** [comment]
> @findepi do you have other comments?

nope, thanks

**Pull request merged**